### ISSUE TITLE
svg styles leaking to other inlined svgs

### ISSUE THREAD

**Issue by mauriciabad** [theGC/html-webpack-inline-svg-plugin#27]
Hi, I found this issue: when inlineing svgs with style tags in them, the styles are being applied to other svgs and a lot of colors are replaced.

I found that adding this to the svgo configuration could fix it:
```js
{
  "plugins":[{
    "inlineStyles":{
      "onlyMatchedOnce":false
  }}]
}
```
Source: https://github.com/svg/svgo/issues/296#issuecomment-380224461

But the structure of the [default config](https://github.com/theGC/html-webpack-inline-svg-plugin/blob/master/svgo-config.js) is different from the one in svgo documentation, so I didn't know how to fix it myself in a PR.

------

And, another thing, I don't understand if the [default config](https://github.com/theGC/html-webpack-inline-svg-plugin/blob/master/svgo-config.js) is applied without doing anything or I have to set it like this:

```js
const svgoConfig = require('html-webpack-inline-svg-plugin/svgo-config');

...

plugins: [
    new HtmlWebpackPlugin({
        template: "./src/index.html", // I also use this option
        filename: "index.html",       // I also use this option
        svgoConfig,
    }),
    new HtmlWebpackInlineSVGPlugin()
]
```

It seems a bit _magic_ that the config is applied to the `HtmlWebpackPlugin` from the `HtmlWebpackInlineSVGPlugin`. A short explanation of how this works could be added to the documentation. 🙂

**Comment by AntonioRedondo** [comment]
Hi @mauriciabad,

I just released v2.3.0 with changes that address your issues:
- Now the custom SVGO configuration for `HtmlWebpackInlineSVGPlugin` follows the same format as SVGO configuration.

   ```js
   plugins: [
       new HtmlWebpackPlugin(),
       new HtmlWebpackInlineSVGPlugin({
           svgoConfig: [
               {
                   removeViewBox: false
               },
               {
                   inlineStyles: {
                       onlyMatchedOnce: false
                   }
               }
           ]
       })
   ]
   ```
- As you can see on the code above SVGO custom configuration is now set inside `HtmlWebpackInlineSVGPlugin({})` instead of `HtmlWebpackPlugin({})`.
- The SVGO defaults for `HtmlWebpackInlineSVGPlugin` are the same as SVGO except the `cleanupIDs` option, which is changed from `true` to `false`. The `HtmlWebpackInlineSVGPlugin` file containing all the defaults has been removed and not they all come directly from SVGO.

Detailed documentation can be read on https://github.com/theGC/html-webpack-inline-svg-plugin#config

Can you tell me if the new changes fix your issues or whether they cause any new one?

**Comment by AntonioRedondo** [comment]
I hope the changes introduced in v2.3.0 fixes the bug.

I'm closing the issue.
